### ISSUE TITLE
Configuring tcp-services-configmap and getting connection refused

### ISSUE THREAD

**Issue by fdvieira** [jcmoraisjr/haproxy-ingress#343]
Hello guys,

I'm doing the following configuration on a TCP config map on HAProxy:

8080: "default/service:8080:PROXY-V2::default/certificate"

So far so good, this works, I can access my service via browser and I can ping it successfully.

But sometimes, when I'm trying to post data on my service via curl I get connection refused. I'm not being able to understand why sometimes it works and other times it doesn't and I get a connection refused. Is this something related to the queue of requests? Do you have any suggestions that may help me?

Thanks a lot.

**Comment by fdvieira** [comment]
Any suggestions?

**Comment by jcmoraisjr** [comment]
Hi, this should be from your backend if everything is properly configured. Do you have logs configured? Is there something useful in the logs that you can share?

**Comment by fdvieira** [comment]
Hello @jcmoraisjr,

Which logs would be useful? From HAproxy? 

While mitigating the issue, I noticed that when I'm configuring TCP services and ports following https://github.com/jcmoraisjr/haproxy-ingress#tcp-services-configmap all my connections are unstable. In this case, I got a connection refused from time to time but in another port that I configured the output is port unreachable and sometimes it tries and connects (it works but this is pretty unstable and I'm not comfortable going production mode).

Any tips?

Thanks a lot.
